mscore/musescore.h

```diff
@@ -684,7 +684,7 @@ class MuseScore : public QMainWindow, public MuseScoreCore {
       void loadFile(const QUrl&);
       QTemporaryFile* getTemporaryScoreFileCopy(const QFileInfo& info, const QString& baseNameTemplate);
       QNetworkAccessManager* networkManager();
-      virtual Score* openScore(const QString& fn, bool switchTab = true, const bool appendToExistingTabs = true);
+      virtual Score* openScore(const QString& fn, bool switchTab = true, const bool appendToExistingTabs = true) override;
```

Comment:
Review needed, it does fix a compiler warning but I'm not sure whether we really want to override `Ms::MuseScoreCore::openScore()` from libmscore/musescoreCore.h line 45 here?

Reply:
Does it matter? I think it is going to be overridden with or without the "override" keyword

Reply:
It was implicitly overridden, is now explcitly overridden, that apeases the compiler. Question is whether this really wanted.

Reply:
I'm just saying even if that wasn't what we want, it still would not be solved (returned to the behaviour of parent class) just by not adding `override`. Am I wrong?

Reply:
No. And that's why I started this discussion here ;-)